modules/storages/app/components/storages/open_project_storage_modal_component.rb

```diff
@@ -48,4 +46,24 @@ def self.dialog_id
   def self.dialog_body_id
     "open-project-storage-modal-body-component"
   end
+
+  def data
+    @data ||= {
```

Comment:
🟢 yeah, nice, lazy building of the data. Did you test it, does the controller comes up correctly? But I think this is quite the improvement to how it was before :D

Reply:
![image](https://github.com/user-attachments/assets/65f8db60-f9db-430f-bd60-710909a5c807)

It does the thing it should do, so, yep.